src/routes/cart/+page.svelte

```diff
@@ -54,8 +54,8 @@
                             id={shoppingCartProductState.id}
                             name={shoppingCartProductState.name}
                             image={shoppingCartProductState.image}
-                            selectedQuantity={shoppingCartProductState.selectedQuantity}
```

Comment:
No it wasn't needed anymore since the logic to increase or decrease the selected quantity is already done in ShoppingCartActions

Reply:
So both the page and the cart item have a dependency on the shoppingCartProductState or we do not store the quantity in the state anymore?

Reply:
we have a component ShoppingCartActions which has a depenency on the cart store. This component is used for now in the ProductCard and ShoppingCartProductCard. In a near future it will be used for the suggested product as well  
<img width="1728" alt="image" src="https://github.com/6C-Solutions-4-Small-Businesses/online-shop-template/assets/46734336/578a6cb2-95a2-4bcb-bb7e-570ad47cfb92">

Reply:
I don't think this is SOLID. Having a state stored and updated from different places will require refactoring anytime we need to use the component in another context (I have been through that in the SearchInput component).
I suggest that the pages be the only one dependent on the state